devicetypes/smartthings/child-switch-health.src/child-switch-health.groovy

```diff
@@ -41,9 +45,6 @@ void off() {
 }
 
 def ping() {
-	parent.ping()
+	// Intentionally left blank as parent should handle this
```

Comment:
Does anything bad happen if we return null here? I don't recall if that's valid or if we need to do something else to make it a noop.